src/pytorch_lightning/trainer/connectors/accelerator_connector.py

```diff
@@ -475,7 +475,7 @@ def _map_deprecated_devices_specific_info_to_accelerator_and_device_flag(
                 if tpu_cores:
                     self._accelerator_flag = "tpu"
                 if gpus:
-                    self._accelerator_flag = "gpu"
+                    self._accelerator_flag = "cuda"
```

Comment:
accidental rename?

Reply:
no that was on purpose :) Soon there will be no direct mapping of an Accelerator and the "gpu" flag anymore :)